cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -176,6 +176,41 @@ func TestValidateIPNetFromString(t *testing.T) {
 	}
 }
 
+func TestValidateAPIEndpoint(t *testing.T) {
+	var tests = []struct {
+		s        *kubeadm.API
```

Comment:
suggest adding a name field of type string, so can have a unique description of each test case.